scripts/jenkins/qa_openstack.sh

```diff
@@ -120,6 +120,9 @@ case "$cloudsource" in
     ;;
 esac
 
+# be sure to use packages from cloud repository
+$zypper dup --no-recommends --from cloud
```

Comment:
how about simply deinstalling cloud-init / paramiko instead? I tend to not like to have  dup in there as imho we should design the repo to not conflict with the existing base operating system.